infrastructure/base/modules/env/main.tf

```diff
@@ -161,6 +172,26 @@ module "backend_cloudrun" {
     {
       name  = "TEST_PUBSUB_SUBSCRIPTION"
       value = module.test_pubsub.subscription_name
+    },
+    {
+      name  = "SMTP_USERNAME"
+      value = "apikey"
+    },
+    {
+      name  = "SMTP_HOST"
+      value = "smtp.sendgrid.net"
+    },
+    {
+      name  = "SMTP_PORT"
+      value = 587
+    },
+    {
+      name  = "MAILER_DEFAULT_HOST"
+      value = var.domain
+    },
+    {
+      name  = "MAILER_DEFAULT_FROM"
+      value = "agnieszka.figiel@vizzuality.com"
```

Comment:
I'd really encourage you to not do this. This seems like the perfect way for this to be forgotten and left like this forever, so I'd suggest you go for something like "noreply@${var.domain}"

Reply:
This needs to be a verified email address (confirmed), and I already verified mine, not keen on spending time sorting out any other email. I will chase the client to replace with actual sender email next week.